nodes/drive_roboclaw.py

```diff
@@ -10,6 +10,8 @@
 from wroboclaw.roboclaw import init_roboclaw
 from wroboclaw.roboclaw.model import Roboclaw, RoboclawChainApi
 
+UINT32_MAX = (1 << 32) - 1
```

Comment:
Should this go to the `util` package somewhere?  I don't know for certain, but now it's defined in two places (mock.py:8), even though it's the exact same constant.

Reply:
Complete as of cb491a0.